packages/safe-core-sdk-types/src/contracts/GnosisSafeContract.ts

```diff
@@ -22,6 +22,6 @@ export interface GnosisSafeContract {
     safeTransaction: SafeTransaction,
     options?: TransactionOptions
   ): Promise<TransactionResult>
-  encode(methodName: any, params: any): string
+  encode(methodName: string, params: any): string
```

Comment:
Can you type this `any`?

Reply:
After discussing this, I'm happy to resolve this with `any`. You may be able to strictly type it via using generics and linking `methodName` to a predefined type of strings. When `T extends <methodNameHere>` you can use a ternary to select the `params`. It's complex so I'll leave it up to you if you want to include it.